README.md

```diff
@@ -1,8 +1,201 @@
+      <td align="center" width="25%">
+        <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/danielcranney/readme-generator/main/public/icons/skills/javascript-colored.svg" width="36" height="36" alt="JavaScript" /></a>
+        <br>JavaScript
+      </td>
+    </tr>
+    <tr>
+      <td align="center" width="25%">
+        <a href="https://reactjs.org/" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/danielcranney/readme-generator/main/public/icons/skills/react-colored.svg" width="36" height="36" alt="React" /></a>
+        <br>React
+      </td>
+      <td align="center" width="25%">
+        <a href="https://nodejs.org/en/" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/danielcranney/readme-generator/main/public/icons/skills/nodejs-colored.svg" width="36" height="36" alt="NodeJS" /></a>
+        <br>NodeJS
+      </td>
+      <td align="center" width="25%">
+        <a href="https://vitejs.dev/" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/danielcranney/profileme-dev/3fc3595593bc992e6febba6580d6c9571f5e625f/public/icons/skills/vite-colored.svg" width="36" height="36" alt="Vite" /></a>
+        <br>Vite
+      </td>
+      <td align="center" width="25%">
+        <a href="https://www.npmjs.com/" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/tandpfun/skill-icons/65dea6c4eaca7da319e552c09f4cf5a9a8dab2c8/icons/Npm-Light.svg" width="36" height="36" alt="npm" /></a>
+        <br>npm
+      </td>
+    </tr>
+    <tr>
+      <td align="center" width="25%">
+        <a href="https://www.ruby-lang.org/" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/devicons/devicon/6910f0503efdd315c8f9b858234310c06e04d9c0/icons/ruby/ruby-original.svg" width="36" height="36" alt="Ruby" /></a>
+        <br>Ruby
+      </td>
+      <td align="center" width="25%">
+        <a href="https://rubyonrails.org/" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/devicons/devicon/6910f0503efdd315c8f9b858234310c06e04d9c0/icons/rails/rails-plain.svg" width="36" height="36" alt="Rails" /></a>
+        <br>Rails
+      </td>
+      <td align="center" width="25%">
+        <a href="https://www.postgresql.org/" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/danielcranney/profileme-dev/3fc3595593bc992e6febba6580d6c9571f5e625f/public/icons/skills/postgresql-colored.svg" width="36" height="36" alt="PostgreSQL" /></a>
+        <br>PostgreSQL
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Contributing
+See [CONTRIBUTING.md](https://github.com/rt4914/KlinicCon-Frontend/blob/main/.github/CONTRIBUTING.md)
+
+## License
+This project is available as open source under the terms of the [MIT License](https://github.com/rt4914/KlinicCon-Frontend/blob/main/LICENSE.txt).
+
+## Contributors
```

Comment:
The contributor list should be in alphabetical order.

Reply:
@aniketpatidar Once you have made this change, you can resolve this specific comment which will allow you to merge this PR.

Reply:
I've updated the contributor list to be in alphabetical order.